pkg/process/exec_conf.go

```diff
@@ -154,6 +157,9 @@ func cleanup(cmd *cobra.Command) {
 	if internalRun == nil {
 		return
 	}
+
+	traceOut := cmd.Flags().String("debug.trace-out", "", "If set, a path to write a process trace SVG to")
```

Comment:
do you think its worthwhile to add this option to the cmds so we can see it as an option? that make sense?

Reply:
is that not what this does?

Reply:
nvm it is there 
```
$ uplink cp --help | grep -i trace
      --debug.trace-out string                 If set, a path to write a process trace SVG to
      --log.stack                   if true, log stack traces
```